middleware/error-handler.js

```diff
@@ -27,7 +27,7 @@ const errorHandlerMiddleware = (err, req, res, next) => {
     customError.statusCode = 404;
   }
 
```

Comment:
Looking above this line, I see wonky indentation. Come through and correct the indentation, or just use a tool like "prettier" to do it for you. If you do want to use the tool but you're not sure how, just reply to this thread and I can help!